src/sentry/api/endpoints/sentry_app_installations.py

```diff
@@ -49,11 +49,16 @@ def post(self, request, organization):
         if not serializer.is_valid():
             return Response(serializer.errors, status=400)
 
+        # check for an exiting installation and return that if it exists
+        slug = serializer.validated_data.get("slug")
+        install_list = SentryAppInstallation.objects.filter(
```

Comment:
I think this could also work
```python
try:
    install = SentryAppInstallation.objects.get(sentry_app__slug=slug, organization=organization)
except SentryAppInstallation.DoesNotExist:
    install = Creator.run(organization=organization, slug=slug, user=request.user, request=request)